### ISSUE TITLE
Bug: Field not showing when key is set to numerical value

### ISSUE THREAD

**Issue by hatsumatsu** [wpninjas/ninja-forms#2961]
Test case:

1. Create a form with a text field and a submit button, embed the form into a page: The field is shown
2. Change the text field's field key to a numerical value like `1`.
The field disappears from the form.

Plugin version: Version 3.1.8
Wordpress version: 4.8.1

**Comment by jameslaws** [comment]
We probably need to add an error check for numeric field keys. Keys should generally follow the same naming conventions of pho functions I believe. Letters and underscores. Numbers only after a letter has first been used. No spaces. That's kind of thing.

**Comment by Much2tall** [comment]
Closing in favor of discussion at https://github.com/wpninjas/ninja-forms/issues/2552
